### ISSUE TITLE
Maximum average reward for half cheetah

### ISSUE THREAD

**Issue by kabirahuja2431** [daggertye/CS294_homework#1]
Hello.
I wanted to know how much maximum reward you were able to get for Half cheetah. I have tried many combinations of hyperparameters but couldn't go beyond 55. My implementation is quite similar to yours, the only big difference is that I have used tensorflow's multivariate Gaussian distribution class to compute the probability (which seemed stupid once I saw your way).

Thanks

**Comment by louaaron** [comment]
Hello, if this is for hw4, I couldn't actually run the homework. The environments that I load in (whether it's their environment or openai's prebuilt one) don't match up with their cost function. 

Sorry

**Comment by kabirahuja2431** [comment]
No need to be sorry. I was actually referring to the hw2. Wanted to know how much average reward you were able to obtain for Half-Cheetah v1 environment.

**Comment by louaaron** [comment]
Oh, I haven't done much experimenting but I think I was able to get around +50. However, it probably could be higher if I tested more variables.

**Comment by kabirahuja2431** [comment]
Okay. The assignment says you should be able to get an average award of 150 by 100 iterations. I tried many things but wasn't able to reach there.
Thanks :+1:
